invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemsSearch/config.js

```diff
@@ -29,6 +29,8 @@ const aggsMappings = {
   status: 'Status',
   medium: 'Medium',
   circulation_status: 'Circulation Status',
+  document_type: 'Document type',
+  keywords: 'Keywords',
```

Comment:
Is `keywords` used on the `ItemsSearch` page?

Reply:
no, it was leftovers after rebase, thanks for spotting that!